espresso-server/app/src/androidTest/java/io/appium/espressoserver/Http/AppiumResponse.java

```diff
@@ -2,21 +2,36 @@
 
 public class AppiumResponse {
     private boolean success;
-    private String message;
+    private String value;
+    private int status;
 
-    public boolean isSuccess() {
+    public AppiumResponse (int status, boolean success, String value) {
+        this.status = status;
+        this.success = success;
+        this.value = value;
+    }
+
+    public boolean isSuccess () {
         return success;
     }
 
-    public void setSuccess(boolean success) {
+    public void setSuccess (boolean success) {
         this.success = success;
     }
 
-    public String getMessage() {
-        return message;
+    public void setStatus (int status) {
+      this.status = status;
```

Comment:
There should be some single rule for java code formatting. I'd propose https://developer.jboss.org/wiki/ImportFormattingRules if you use Android Studio (IDEA fork)